example/notify_inval_entry.c

```diff
@@ -228,7 +230,15 @@ static void tfs_readdir(fuse_req_t req, fuse_ino_t ino, size_t size,
     }
 }
 
+static void tfs_init(void *userdata, struct fuse_conn_info *conn) {
+    if(options.only_expire && !(conn->capable & FUSE_CAP_EXPIRE_ONLY)) {
+        fprintf(stderr, "FUSE_CAP_EXPIRE_ONLY not supported by kernel\n");
+        fuse_session_exit(*(struct fuse_session **) userdata);
+    }
+}
+
 static const struct fuse_lowlevel_ops tfs_oper = {
+    //.init	= tfs_init,
```

Comment:
I'm a bit confused. Why are we adding this function only to comment it out? Let's either not add it in the first place, or add it and use it.

(I think for the purposes of the example, it does not matter much if we are failing at init() time or when attempting to call the function)

Reply:
the `tfs_init()` was added by `@matthiasgoergens` during his fix of this example. So I thought I better let it in here until we decide where to check for the support. Personally I would prefer to check inside the loop to see that the function really fails

But I am open also to just adding in `tfs_init`

Reply:
ok for whatever reason the comments were deleted...

`tfs_init` was the new function `@matthiasgoergens` provided to fix this example. i would rather prefer to check not in the init but in the function itself to see that the correct return type is given. 

but i'm open to whatever you prefer!

Reply:
If memory serves right, I added an init function to fix a memory leak or so.

Reply:
your `tfs_init` only checks if `expire` is supported. otherwise it aborts the program.